src/kernel_introspection/k8s.cr

```diff
@@ -38,24 +38,25 @@ module KernelIntrospection
         cmdline 
       end
 
-      def self.verify_single_proc_tree(original_parent_pid, name, proctree : Array(Hash(String, String)))
+      def self.verify_single_proc_tree(original_parent_pid, name, proctree : Array(Hash(String, String)), excluded_processes : Array(String)? = nil)
```

Comment:
"excluded_processes = [] of String" would allow removing "excluded_processes" check in line 51.

Reply:
done